docs/quickstart.md

```diff
@@ -53,7 +53,7 @@ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/
 ##### Add repository
 
 ```shell
-helm repo add actions-runner-controller https://actions-runner-controller.github.io/actions-runner-controller
+helm repo add actions https://actions-runner-controller.github.io/actions-runner-controller
```

Comment:
The alternative to this change, in order to make this command work with the next one without the proposed change, would be to change line 65 below to:
`--wait actions-runner-controller actions-runner-controller/actions-runner-controller`

Reply:
Looks like the issue was corrected by updating line 65 in another PR so this change isn't necessary